presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergUtil.java

```diff
@@ -431,6 +419,7 @@ private static boolean isValidPartitionType(com.facebook.presto.common.type.Type
                 DOUBLE.equals(type) ||
                 DATE.equals(type) ||
                 TIMESTAMP.equals(type) ||
+                TIME.equals(type) ||
```

Comment:
Can you separate the bug fixes related to time/timestamp to a separate commit?

Reply:
Sure.